terraform/environments/analytical-platform-compute/s3-buckets.tf

```diff
@@ -21,3 +21,70 @@ module "mlflow_bucket" {
 
   tags = local.tags
 }
+
+module "mojap_derived_tables_replication_bucket" {
+  #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
+  #checkov:skip=CKV_TF_2:Module registry does not support tags for versions
+
+  source  = "terraform-aws-modules/s3-bucket/aws"
+  version = "4.2.1"
+
+  bucket = "mojap-compute-${local.environment}-derived-tables-replication"
+
+  force_destroy = true
+
+  object_lock_enabled = false
+
+  versioning = {
+    status = "Disabled"
+  }
+
+  server_side_encryption_configuration = {
+    rule = {
+      bucket_key_enabled = true
+      apply_server_side_encryption_by_default = {
+        kms_master_key_id = module.mojap_derived_tables_replication_s3_kms.key_arn
+        sse_algorithm     = "aws:kms"
+      }
+    }
+  }
+
+  logging = {
+    target_bucket = module.mojap_compute_logs_bucket.s3_bucket_id
+    target_prefix = "mojap-derived-tables-replication/"
+  }
+
+  tags = local.tags
+}
+
+module "mojap_compute_logs_bucket" {
+  #checkov:skip=CKV_TF_1:Module registry does not support commit hashes for versions
+  #checkov:skip=CKV_TF_2:Module registry does not support tags for versions
+
+  source  = "terraform-aws-modules/s3-bucket/aws"
+  version = "4.2.1"
+
+  bucket = "mojap-compute-${local.environment}-logs"
+
+  force_destroy = false
+
+  object_lock_enabled = false
+
+  versioning = {
+    status = "Disabled"
+  }
+
+  server_side_encryption_configuration = {
+    rule = {
+      bucket_key_enabled = true
+      apply_server_side_encryption_by_default = {
+        kms_master_key_id = module.mojap_compute_logs_s3_kms.key_arn
+        sse_algorithm     = "aws:kms"
+      }
+    }
+  }
+
+  tags = local.tags
+
+  policy = data.aws_iam_policy_document.s3_server_access_logs_policy.json
```

Comment:
Move this upwards, `tags` is generally always last, maybe after `force_destroy`?